### ISSUE TITLE
Wmstats site report

### ISSUE THREAD

**Issue by prozober** [dmwm/WMCore#7427]
I've seen some weird reports in wmstats for a couple of ACDCs.
E.g. https://cmsweb.cern.ch/wmstatsserver/data/jobdetail/prozober_ACDC0_Run2016H-v1-HLTPhysicsIsolatedBunch-09Nov2016_8023_161121_230720_1955
The report says some jobs went to jobcooloff at T0_CH_CERN, but in the state_history we have 'location": "T1_US_FNAL'.
Is there something funny to be fixed?
Thanks.

**Comment by amaltaro** [comment]
Likely using information from different sources for this report. Just in case, here is the interesting piece of report:
```
          "T0_CH_CERN": {
            "errorCount": 5, 
            "samples": [
              {
                "agent_name": "vocms0309.cern.ch", 
                "state_history": [
                  {
                    "newstate": "jobcooloff", 
                    "oldstate": "jobfailed", 
                    "location": "T1_US_FNAL", 
                    "timestamp": 1479957639
                  }
                ], 
                "inputfiles": [], 
                "errors": {
                  "JobKilled": [
                    {
                      "type": "JobKilled", 
                      "details": "The job was killed by the WMAgent for using too much wallclock time (WMAgent) Job status was Running.", 
                      "exitCode": 71304
                    }
                  ]
                }, 
                "workflow": "prozober_ACDC0_Run2016H-v1-HLTPhysicsIsolatedBunch-09Nov2016_8023_161121_230720_1955", 
                "timestamp": 1479957639, 
                "_rev": "3-30fddeae2490f7167cf70026ec57bfd7", 
                "site": "T0_CH_CERN", 
                "lumis": [], 
                "state": "jobcooloff", 
                "cms_location": {}, 
                "outputdataset": null, 
                "acdc_url": "https://cmsweb.cern.ch/couchdb/acdcserver", 
                "output": [], 
                "wmbsid": 2448379, 
                "retrycount": 0, 
                "_id": "3fa3fb8a-b062-11e6-a094-001e67abf094-132", 
                "type": "jobsummary", 
                "task": "/prozober_ACDC0_Run2016H-v1-HLTPhysicsIsolatedBunch-09Nov2016_8023_161121_230720_1955/DataProcessing", 
                "jobtype": "Processing", 
                "exitcode": 71304
              }
```

I'll leave it to Seangchan because he knows it much better than me :-D

**Comment by amaltaro** [comment]
This should have been fixed by #8247
